src/env.ts

```diff
@@ -1,6 +1,20 @@
 import * as envalid from 'envalid'
 import { singleton } from 'tsyringe'
 
+const strArrayValidator = envalid.makeValidator((input) => {
+  const arr = input
+    .split(',')
+    .map((s) => s.trim())
+    .filter((s) => !!s)
```

Comment:
mmm... not sure but my brain wants to tell me something about `bang bang` operator, but can not remember from top oh my head under which circumstances it could not work as expected, but knowing that we have tests and etc pretty sure it will get flagged.

Reply:
we're guarenteed to have a string at this point so `!!` will remove falsey strings, notable `''`